easybuild/tools/systemtools.py

```diff
@@ -106,6 +107,28 @@ def get_core_count():
     _log.nosupport("get_core_count() is replaced by get_avail_core_count()", '2.0')
 
 
+def get_total_memory():
+    """
+    Try to ascertain this node's total memory
+    
+    @return: total memory as an integer, specifically a number of megabytes
```

Comment:
If possible, it would be nice, if there was the ability to tune things from the outside about this, such as $BC_MEM_PER_NODE:
http://centers.hpc.mil/consolidated/bc/policies.php?choice=environment

The meaning of this is to be able to modify builds behavior dynamically, by adjusting such a variable.